sr_data_visualization/src/sr_data_visualization/sr_data_visualizer_gui.py

```diff
@@ -207,5 +202,5 @@ def shutdown_plugin(self):
     rospy.init_node("sr_data_visualizer")
     app = QApplication(sys.argv)
     data_visualiser_gui = SrDataVisualizer(None)
-    data_visualiser_gui._widget.show()
+    data_visualiser_gui._widget.show()  # pylint: disable=W0212
```

Comment:
create getter for _widget

Reply:
can't be done as its part of QT's default library

Reply:
https://github.com/shadow-robot/sr-visualization/blob/119480ceeff7a1588d2edf0e555fed0f9fff7ba2/sr_data_visualization/src/sr_data_visualization/sr_data_visualizer_gui.py#L78 it's defined within the SrDataVisualizer class, so it seems we could creat a getWidget() / getMainWidget() or sth like this method?